integration/entity.go

```diff
@@ -60,7 +70,25 @@ func (e *Entity) AddCommonDimension(key string, value string) {
 	e.lock.Lock()
 	defer e.lock.Unlock()
 
-	e.CommonDimensions[key] = value
+	e.CommonDimensions.Attributes[key] = value
+}
+
+// AddCommonTimestamp adds a new common timestamp to the entity.
+func (e *Entity) AddCommonTimestamp(timestamp time.Time) {
+	e.lock.Lock()
+	defer e.lock.Unlock()
+
+	t := timestamp.Unix()
+	e.CommonDimensions.Timestamp = &t
+}
+
+// AddCommonInterval adds a new common timestamp to the entity from duration in nanoseconds.
+func (e *Entity) AddCommonInterval(timestamp time.Duration) {
+	e.lock.Lock()
+	defer e.lock.Unlock()
+
+	t := timestamp.Milliseconds()
```

Comment:
comment above says nanoseconds. Is this ok? ☝️

Reply:
The function parameter is in nanoseconds, the resulting interval in milliseconds.

New commit changed the comment as it was misleading. 

Thanks!

Reply:
got it. Thanks!!